internal/billing_test.go

```diff
@@ -81,62 +81,84 @@ func TestMwsAccLogDelivery(t *testing.T) {
 }
 
 func TestMwsAccBudgets(t *testing.T) {
+	// Currently, the API returns errors
```

Comment:
Not only for the SDK. Even curl returns 400 when using Oauth tokens. Investigation ongoind.